notebooks/data analysis/Pathfinder_system_abilities_attributes.ipynb

```diff
@@ -0,0 +1,1915 @@
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "display(hardness)"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Nie wiem co się stało przy rozpakowywaniu, ale tu są tylko *values*\n",
+    "\n",
+    "*Wydaje mi się, że to też można użyć*\n",
+    "\n",
+    "Wartość specyficzna - głównie dla obiektów -> możliwe, iż powyższe zdanie jest nieaktualne"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "### Analiza attributes: *adjustment*"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 250,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "0   NaN\n1   NaN\n2   NaN\n3   NaN\n4   NaN\nName: adjustment, dtype: float64"
+     },
+     "execution_count": 250,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attributes.adjustment.head()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 251,
+   "outputs": [
+    {
+     "data": {
+      "text/plain": "Series([], Name: adjustment, dtype: float64)"
+     },
+     "execution_count": 251,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "attributes[attributes.adjustment.notnull()].adjustment"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 252,
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "<class 'pandas.core.series.Series'>\n",
+      "Int64Index: 1206 entries, 0 to 364\n",
+      "Series name: adjustment\n",
+      "Non-Null Count  Dtype  \n",
+      "--------------  -----  \n",
+      "0 non-null      float64\n",
+      "dtypes: float64(1)\n",
+      "memory usage: 51.1 KB\n"
+     ]
+    }
+   ],
+   "source": [
+    "attributes.adjustment.info()"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "Tutaj są same nulle, więc można usunąć"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "markdown",
+   "source": [
+    "## Podsumowując:\n",
```

Comment:
Po co ja się tak namęczyłam jak tu jest takie ładne podsumowanie.

Reply:
To ja mam taki pomysł, żeby dodać kategorię na potem i wrzucić tam takie dane które mogłyby mieć znaczenie ale są trochę bardziej upierdliwe do uwzględnienia i w razie czego kiedyś je dodamy.